content/tutorials/guides/how-to-collect-product-variations.md

```diff
@@ -0,0 +1,274 @@
+:::
+
+
+## Collect product variations
+
+### Workflow
+
+![schema_product_models](../../img/tutorials/how-to-collect-product-variations/schema_product_models.svg)
+
+In this tutorial, we will introduce you to the two use cases you may encounter for retrieving products with variations.
+
+- **Use case 1**: Collect all product variation information, just as they are in the PIM (up to 2 possible levels)
+- **Use case 2**: Collect all product variation information on 1 level only)
+
+::: info
+We noticed that many e-commerce solutions understand product variation on only one level. This means that for Akeneo a special recollection of the variations must be done to have them all on the same level.
+:::
+
+### 0 - Initialization
+
+```php [activate:PHP]
+
+$pimUrl = 'https://url-of-your-pim.com';
+$appToken = 'your_app_token'; // Token provided during oAuth steps
+
+// If you haven't done it yet, please follow the Guzzle official documentation for installing the client 
+// https://docs.guzzlephp.org/en/stable/overview.html#installation
+
+// Set your client for querying Akeneo API as follows
+$client = new \GuzzleHttp\Client([
+    'base_uri' => $pimUrl,
+    'headers' => ['Authorization' => 'Bearer ' . $appToken],
+]);
+```
+
+### Use case 1: Collect product variation information - all levels
+
+#### 1. Collect product models
+##### 1.1 You are following the App workflow?
+
+In the guided tutorial How to get families and attributes, we have stored a family_code_list. It’s time to use it!
+
+```php [activate:PHP]
+
+function getProductModels(): array
+{
+    $client = getApiClient();
+
+    $maxProductsPerPage = 100;
+    $maxFamiliesPerQuery = 3;
+    $locales = 'fr_FR';
+    $scope = 'ecommerce';
+
+    $familyCodeChunks = array_chunk(getFamilyCodes(), $maxFamiliesPerQuery);
+
+    $apiUrl = '/api/rest/v1/product-models?'
+        . 'locales=%s'
+        . '&scope=%s'
+        . '&search={"family":[{"operator":"IN","value":%s}]}'
+        . '&limit=' . $maxProductsPerPage;
+
+
+    // Collect product models from paginated API
+    $productModels = [];
+    foreach ($familyCodeChunks as $familyCodes) {
+        $response = $client->get(sprintf($apiUrl, $locales, $scope, json_encode($familyCodes)));
+        $data = json_decode($response->getBody()->getContents(), true);
+        $productModels[] = $data['_embedded']['items'];
+    }
+
+    $productModels = array_merge(...$productModels);
+
+    saveProductModels($productModels);
+
+    return $productModels;
+}
+
+```
+
+##### 1.2 You are not following the App workflow?
+Simply get the attribute type by requesting the API
+
+```php [activate:PHP]
+
+function getProductModels(): array
+{
+    $client = getApiClient();
+
+    $maxProductsPerPage = 100;
+    $locales = 'fr_FR';
+    $scope = 'ecommerce';
+
+    $apiUrl = '/api/rest/v1/product-models?'
+        . 'locales=%s'
```

Comment:
No locales --> https://www.notion.so/akeneo/Guided-tutorial-How-to-collect-product-variations-183f7b14a75147788336d48fbcaf7c7f